recipes/yasm/all/conanfile.py

```diff
@@ -71,6 +71,10 @@ def _generate_autotools(self):
     def _generate_cmake(self):
         tc = CMakeToolchain(self)
         tc.cache_variables["YASM_BUILD_TESTS"] = False
+        # Don't build shared libraries because:
+        # 1. autotools doesn't build shared libs either
+        # 2. the shared libs don't support static libc runtime (MT and such)
+        tc.cache_variables["BUILD_SHARED_LIBS"] = False
```

Comment:
Shouldn't this go in the validate method? (i only quickly skimmed the mentioned issue - trying to help but limited on time)

Reply:
If somebody needs to link to shared libraries, then that later person can add this.
The comment explains why, so the fix is easy.

Reply:
there is no shared option in yasm recipe anyway. The lib is ignored (it could be removed after packaging for autotools also by the way).

Reply:
I added that.

Reply:
Thanks for filling me in 💟